LANGUAGES/languages.json

```diff
@@ -360,51 +360,51 @@
         "name":"Turkish (Türkçe)",
         "FontFamily":"Courier New",
         "FontSize":16,
-        "Author":"Erhan Kültür/crasadure",
+        "Author":"Erhan Kültür/crasadure, Oğuz Ersen",
         
         "STR_ERROR":"Hata",
         "STR_WARNING":"Uyarı",
         "STR_INFO":"Bilgi",
         "STR_INCORRECT_DIR":"Lütfen doğru dizin altında çalıştırın!",
-        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırma, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
+        "STR_INCORRECT_TREE_DIR":"Dosyayı burada çalıştırmayın, lütfen yayınlanan en son kurulum paketini indirin ve oradan çalıştırın.",
         "STR_DEVICE":"Aygıt",
         "STR_LOCAL_VER":"Ventoy GÜNCEL Sürüm",
-        "STR_DISK_VER":"Cihazdaki Ventoy Sürümü",
+        "STR_DISK_VER":"Aygıttaki Ventoy Sürümü",
         "STR_STATUS":"Durum - HAZIR",
         "STR_INSTALL":"Kur",
         "STR_UPDATE":"Güncelle",
-        "STR_UPDATE_TIP":"Ventoyun versiyon bilgisi güncellenecek, ISO dosyalarınız bundan etkilenmiyecek.#@Devam edilsinmi?",
-        "STR_INSTALL_TIP":"Disk biçimlendirilecek ve tüm veriler kaybolacak.#@Devam edilsinmi?",
-        "STR_INSTALL_TIP2":"Disk biçimlendirilecek ve tüm veriler kaybolacak.#@Devam edilsinmi? (Son UYARI!)",
-        "STR_INSTALL_SUCCESS":"Tebrikler!#@ventoy cihaza başarıyla kuruldu.",
-        "STR_INSTALL_FAILED":"Yükleme sırasında bir hata oluştu. USB'yi yeniden takıp tekrar deneyebilirsiniz. Ayrıntılar için log.txt dosyasına bakın.",
-        "STR_UPDATE_SUCCESS":"Tebrikler!#@ventoy cihazda başarıyla güncellendi.",
+        "STR_UPDATE_TIP":"Yükseltme işlemi güvenlidir, ISO dosyalarınız bundan etkilenmeyecek.#@Devam edilsin mi?",
+        "STR_INSTALL_TIP":"Disk biçimlendirilecek ve tüm veriler kaybolacak.#@Devam edilsin mi?",
+        "STR_INSTALL_TIP2":"Disk biçimlendirilecek ve tüm veriler kaybolacak.#@Devam edilsin mi? (Son UYARI!)",
+        "STR_INSTALL_SUCCESS":"Tebrikler!#@Ventoy aygıta başarıyla kuruldu.",
+        "STR_INSTALL_FAILED":"Kurulum sırasında bir hata oluştu. USB'yi yeniden takıp tekrar deneyebilirsiniz. Ayrıntılar için log.txt dosyasına bakın.",
+        "STR_UPDATE_SUCCESS":"Tebrikler!#@Ventoy aygıtta başarıyla güncellendi.",
```

Comment:
"etkilenmiyecek" ve "edilsinmi" kelimelerindeki yazım yanlışları düzeltildi. Tutarlı olması için "installation" sözcüğü "yükleme" yerine "kurulum" olarak değiştirildi. Büyük/küçük harf düzeltmeleri yapıldı.

Reply:
Madem tam doğrusunu yapacaksak; 381.satırda şu şekilde düzeltirmisin?
Ventoy,aygıtta başarıyla güncellendi.
379.satırda ise;
Ventoy,aygıta başarıyla kuruldu...
Yukarıdaki iki satırda Virgül(,) karakterini kullanmak gerek diye düşünüyorum

Reply:
Düzelttim.